Tasks/string_builder.py

```diff
@@ -0,0 +1,52 @@
+class StringBuilder:
+
+    def __init__(self, data):
+        self.string = list(data)
+
+    def append(self, elem: str):
+        self.string.extend(list(elem))
+
+    def __str__(self):
+        return ''.join(self.string)
+
+    def insert(self, elem: str, ind: int = 0):
+        if ind > len(''.join(self.string)):
+            raise IndexError('Index out of range')
+        if ind < 0:
+            self.string.insert(ind, elem)
+        else:
+            self.string.insert(ind, elem)
+
+    def __getitem__(self, key: int or slice):
+        if isinstance(key, slice):
+            start, stop, step = key.indices(len(self.string))
+            return ''.join(str(i) for i in self.string[start:stop:step])
+        if isinstance(key, int):
+            if key > len(self.string):
+                raise IndexError('Index out of range')
+            if key < 0:
+                key += len(self.string)
+            return ''.join(str(i) for i in self.string)[key]
```

Comment:
```suggestion
            return str(self.string[key])
```
Maybe just this if you only return N-th element? It will be similar from the list and string